src/constants/network.ts

```diff
@@ -1,6 +1,6 @@
 export const networks = ["local", "mainnet", "matic", "maticmum", "goerli", "sepolia"] as const;
 
-export type networkName = typeof networks[number];
+export type networkName = (typeof networks)[number];
```

Comment:
FYI: this changes is the new eslint-prettier want me to make if not will have the ugly red line and can't build. and i dont want to add the //disable-eslint-next-line, feel that it is unnecessary.